tests/aws/stepfunctions/v2/scenarios/test_base_scenarios.py

```diff
@@ -177,6 +177,26 @@ def test_map_state_catch(
             exec_input,
         )
 
+    def test_map_state_catch_empty_fail(
```

Comment:
small nit: all tests in `tests/aws/*` will now need to have one of the `@markers.aws.*`